src/main/java/ch/njol/skript/expressions/ExprCommandInfo.java

```diff
@@ -43,108 +50,106 @@
 
 @Name("Command Info")
 @Description("Get information about a command.")
-@Examples({"main name of command \"skript\"",
+@Examples({
+	"main name of command \"skript\"",
 	"description of command \"help\"",
 	"label of command \"pl\"",
 	"usage of command \"help\"",
 	"aliases of command \"bukkit:help\"",
 	"permission of command \"/op\"",
 	"command \"op\"'s permission message",
-	"command \"sk\"'s plugin owner"})
+	"command \"sk\"'s plugin owner",
+	"",
+	"command /greet <player>:",
+		"\tusage: /greet <target>",
+		"\ttrigger:",
+			"\t\tif arg-1 is sender:",
+				"\t\t\tsend \"&cYou can't greet yourself! Usage: %the usage%\"",
+				"\t\t\tstop",
+			"\t\tsend \"%sender% greets you!\" to arg-1",
+			"\t\tsend \"You greeted %arg-1%!\""
+})
 @Since("2.6")
 public class ExprCommandInfo extends SimpleExpression<String> {
 
 	private enum InfoType {
-		NAME,
-		DESCRIPTION,
-		LABEL,
-		USAGE,
-		ALIASES,
-		PERMISSION,
-		PERMISSION_MESSAGE,
-		PLUGIN,
+		NAME(Command::getName),
+		DESCRIPTION(Command::getDescription),
+		LABEL(Command::getLabel),
+		USAGE(Command::getUsage),
+		ALIASES(null), // Handled differently
+		PERMISSION(Command::getPermission),
+		PERMISSION_MESSAGE(Command::getPermissionMessage),
+		PLUGIN(command -> {
+			if (command instanceof PluginCommand) {
+				return ((PluginCommand) command).getPlugin().getName();
+			} else if (command instanceof BukkitCommand) {
+				return "Bukkit";
+			} else if (command.getClass().getPackage().getName().startsWith("org.spigot")) {
+				return "Spigot";
+			} else if (command.getClass().getPackage().getName().startsWith("com.destroystokyo.paper")) {
+				return "Paper";
+			}
+			return "Unknown";
+		});
+
+		private final Function<Command, String> function;
+
+		InfoType(Function<Command, String> function) {
+			this.function = function;
+		}
+
 	}
 
 	static {
-		Skript.registerExpression(ExprCommandInfo.class, String.class, ExpressionType.SIMPLE,
-			"[the] main command [label] of command %strings%", "command %strings%'[s] main command [name]",
-			"[the] description of command %strings%", "command %strings%'[s] description",
-			"[the] label of command %strings%", "command %strings%'[s] label",
-			"[the] usage of command %strings%", "command %strings%'[s] usage",
-			"[(all|the|all [of] the)] aliases of command %strings%", "command %strings%'[s] aliases",
-			"[the] permission of command %strings%", "command %strings%'[s] permission",
-			"[the] permission message of command %strings%", "command %strings%'[s] permission message",
-			"[the] plugin [owner] of command %strings%", "command %strings%'[s] plugin [owner]");
+		Skript.registerExpression(ExprCommandInfo.class, String.class, ExpressionType.PROPERTY,
```

Comment:
breaking change as your changing the expression loading order to be 2 spots lower than SIMPLE.